Parsers/ASimDns/Tests/Microsoft_DNS Server_Dns_SchemaTest_ASimDnsMicrosoftNXLog.csv

```diff
@@ -0,0 +1,118 @@
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcOriginalRiskLevel]"
+"(2) Info: Missing optional field [SrcOriginalUserType]"
+"(2) Info: Missing optional field [SrcProcessGuid]"
+"(2) Info: Missing optional field [SrcProcessId]"
+"(2) Info: Missing optional field [SrcProcessName]"
+"(2) Info: Missing optional field [SrcRiskLevel]"
+"(2) Info: Missing optional field [SrcUserAWSId]"
+"(2) Info: Missing optional field [SrcUserAadId]"
+"(2) Info: Missing optional field [SrcUserId]"
+"(2) Info: Missing optional field [SrcUserOktaId]"
+"(2) Info: Missing optional field [SrcUserScopeId]"
+"(2) Info: Missing optional field [SrcUserScope]"
+"(2) Info: Missing optional field [SrcUserSessionId]"
+"(2) Info: Missing optional field [SrcUserSid]"
+"(2) Info: Missing optional field [SrcUserType]"
+"(2) Info: Missing optional field [SrcUserUid]"
+"(2) Info: Missing optional field [SrcUsername]"
+"(2) Info: Missing optional field [TenantId]"
+"(2) Info: Missing optional field [ThreatCategory]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatField]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIpAddr]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatName]"
+"(2) Info: Missing optional field [ThreatOriginalConfidence]"
+"(2) Info: Missing optional field [ThreatOriginalRiskLevel]"
+"(2) Info: Missing optional field [ThreatRiskLevel]"
+"(2) Info: Missing optional field [UrlCategory]"
+"(2) Info: Missing recommended alias [Hostname] aliasing non-existent column [SrcHostname]"
+"(2) Info: extra unnormalized column [Category]"
+"(2) Info: extra unnormalized column [EventReceivedTime_t]"
+"(2) Info: extra unnormalized column [Level]"
+"(2) Info: extra unnormalized column [ProviderGuid_g]"
+"(2) Info: extra unnormalized column [QueryTypeName]"
+"(2) Info: extra unnormalized column [QueryType]"
+"(2) Info: extra unnormalized column [Query]"
+"(2) Info: extra unnormalized column [ResponseCodeName]"
+"(2) Info: extra unnormalized column [ResponseCode]"
+"(2) Info: extra unnormalized column [_ResourceId]"

```

Comment:
@jszigetvari-nxlog Please project-away that are showing as unnormalized. i.e. from line 109 to 118. Please include these at line 327 where we are doing project-away for other fields.

@jszigetvari-nxlog  My bad, I would like to correct my previous suggestion of keep outdated fields, I think we can safely remove these old fields since these are no longer are part of DNS schema as detected by ASIM tester too. Please remove them from both ASIM and VIM files.

Reply:
Please make sure to apply any changes on both ASIM and VIM files.

Reply:
done

Reply:
@jszigetvari-nxlog  when I ran the parser I got error for Category, Level fields not existing, can you please have a look?

Reply:
@vakohl Yes, I removed them intentionally, because they showed up as a finding in the schema tests:
```
"(2) Info: extra unnormalized column [Category]"
"(2) Info: extra unnormalized column [EventReceivedTime_t]"
"(2) Info: extra unnormalized column [Level]"
```
But I can undo those changes if you would like.

Reply:
After removing the Category, Level, I wasn't getting any error. Please remove these two from Project-Away in both parsers

Reply:
sure thing. fixed.